po/extra/lt.po

```diff
@@ -0,0 +1,302 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-10-05 21:43+0900\n"
+"PO-Revision-Date: 2022-11-01 22:33+0200\n"
+"Last-Translator: PolPolyLingo <paulius.narkevicius@vilniustech.lt>\n"
+"Language-Team: \n"
+"Language: lt\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && (n%100<11 || n%100>19) ? 0 : n"
+"%10>=2 && n%10<=9 && (n%100<11 || n%100>19) ? 1 : 2);\n"
+"X-Generator: Poedit 3.1.1\n"
+
+#: data/pinit.appdata.xml.in:8 data/pinit.desktop.in:3
+msgid "Pin It!"
+msgstr "Pakišk tai!"
```

Comment:
[Question]
Actually I was not intended to be translated this string (I should have been explicit that in a comment though), but would it sound more natural to translate this than keeping it as it is in your language?
(If so, I'll mark the same text in elsewhere in this app translatable.)

Reply:
There number of places, in instance Streaming Service of the ...Tube which let us see some Headlines in the Natural Language, even if the Contents it'self not in Natural Language. I will leave decision on this for You thou.

Reply:
Hmm, it seems like I wasn't able to explain it enough. The source string "Pin It!" refers to the application name, which is a proper noun (the name of "…Tube" in your comment, for example). This is the reason I don't expect this string to be translated. However, I also wonder some language(s) might do translate this kind of noun and if so it's fine to translate this text. These are what I wanted to tell you. :bowing_woman:

Reply:
If it would be only proper noun, then the name of the App I would assume only the "Pin".
"Pin It!" already tells us in instance to take some sharp thing, let us say, the Tail and to attach it into the right place. 
Well You the Boss!

Reply:
Then for more correct way rolling back to the English would be good.